client-libs/typescript/src/openai/mergeChunks.ts

```diff
@@ -13,7 +14,7 @@ const omit = <T extends Record<string, unknown>, K extends keyof T>(
 
 export default function mergeChunks(
   base: ChatCompletion | null,
-  chunk: ChatCompletionChunk,
+  chunk: ChatCompletionChunk & { usage?: CompletionUsage },
```

Comment:
Both Anyscale and Fireworks have a usage property that is not compatible with OpenAI chunks. Should we introduce a custom type for chunks?

Reply:
I think returning usage is fine but I probably wouldn't document that we do, since we might add more providers in the future that stick closer to the OpenAI spec and don't include it.